lib/routes/njupco/handler-func.ts

```diff
@@ -0,0 +1,53 @@
+import * as cheerio from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+
+export async function handler() {
+    interface NewsItem {
+        title: string,
+        link: string,
+        pudDate: Date | null,
+        description: string,
+    }
```

Comment:
Use https://github.com/DIYgod/RSSHub/blob/b98b4977ff196efbc1d27b8cf771d5c1e407f935/lib/types.ts#L32 instead of reinvent types.